packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.v1.vue

```diff
@@ -152,6 +157,10 @@ export default {
             type: Boolean,
             default: true
         },
+        cuisines: {
+            type: Array,
+            default: null
```

Comment:
I'd suggest making the default value an empty array and therefore allow simplifying the in-template condition (just use `cuisines.length`) on line 40:
```suggestion
            default: () => []
```

Reply:
Yeah I agree with Ben!